src/js/components/ErrorsAlert.js

```diff
@@ -47,7 +47,7 @@ const ErrorsAlert = function(props) {
   const errorItems = errorMessages.map((message, index) => {
     return (
       <li key={index} className="errorsAlert-listItem">
-        {message}
+        {message}.
```

Comment:
Why are we adding a period when we didn't have one originally?

Reply:
From the trade-offs section:
```


Added a dot to the end of the messages here for consistency - https://github.com/dcos/dcos-ui/pull/3827/files#diff-6b3adbaa5ad3b79d147e929059c70c0cR50 . However, in Chinese the dot is different, so I wonder if we should have a dot in the message catalog.
```
Some errors had a dot and some didn't, so I added the dot here so that all error have it.